src/_types.ts

```diff
@@ -6,6 +6,15 @@ export type PredIndexedOptional<T, K> = (
   array?: Array<T>
 ) => K;
 
+/**
+ * This helper type is useful to unwrap complex types exposing the underlying
+ * bare type. For example `Omit<{a: number, b: string}, 'a'>` becomes
+ * `{b: string}` directly.
+ */
+export type Unwrap<T> = {
+  [K in keyof T]: T[K];
+} & NonNullable<unknown>;
+
```

Comment:
It looks like you are implementing type-fest's Simplify. https://github.com/sindresorhus/type-fest/blob/main/source/simplify.d.ts

I'd keep the same name and maybe add a link to that project so we can sync any updates that are done to it from that project in the future.
```suggestion
/**
 * This helper type is useful to unwrap complex types exposing the underlying
 * bare type. For example `Omit<{a: number, b: string}, 'a'>` becomes
 * `{b: string}` directly.
 */
export type Simplify<T> = {
  [K in keyof T]: T[K];
} & NonNullable<unknown>;

```

Reply:
I removed the type altogether